README.md

```diff
@@ -1,4 +1,12 @@
-# ChimeraOS
+<!--Author: D.A.Pelasgus-->
+<p align="center"><img src="assets/logo.svg" alt="ChimeraOS" style="width: 150px;" /></p>
+
+[![status](https://img.shields.io/badge/status-stable-%23961937.svg?style=for-the-badge)](https://github.com/chimeraos/install-media/releases/latest)
+[![License](https://img.shields.io/badge/License-MIT-%23961937.svg?style=for-the-badge)](https://chimeraos.org)
```

Comment:
```suggestion
[![License](https://img.shields.io/badge/License-MIT-%23961937.svg?style=for-the-badge)](https://github.com/ChimeraOS/chimeraos/blob/master/LICENSE)
```

Already linking to the website, why not link to the license on the license badge?